Spell/__init__.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+"""
+Check spelling of any word in any language you want. Example: spell en great
+"""
+
+from albertv0 import Item
+from albertv0 import ClipAction
+import os
+import re
+import subprocess
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Spell checker"
+__version__ = "2.0"
+__trigger__ = "spell "
+__author__ = "Marek Mazur"
+__dependencies__ = []
+
+
+dict_file = '~/.local/share/albert/org.albert.extension.python/modules/Spell/dictionaries/{language}.dict'
```

Comment:
is it a good idea to rely on the shared data of another extension? i'm not sure how the dictionaries go into org.albert.extension.python/... anyway?

Reply:
another extension? it is for this extension only. Dictonaries must prepared manually as I described here https://github.com/marek-mazur/albert-spell-extension/blob/master/README.md

Reply:
Hmm... But there is no README.md file in this PR. Should i add it or add instructions to dock block?

Reply:
ok, i didn't see the README.
still, i thought the org.albert.extension.python directory is for the *python* extension of albert. i thought another extension would have an org.albert.extension.otherext folder. but sorry if i'm wrong, didn't read the spec lately.

Reply:
Yep. This repository shouldn't be named "python". Better idea is rename it to extensions and totally deprecate old extension repository... @ManuelSchneid3r Will you consider that to prevent mistakes like this in future?

Reply:
Somewhere in the faar future other languages may be added. Therefore *extensions* would be ambiguous.


> i thought the org.albert.extension.python directory is for the python extension of albert.

it is. i think relative paths would be better to access resources shipped with the python module. 

persisitent user data should go to $(dataLocation())/<plugin_id>/ e.g. ~/.local/share/albert/spell/

spell is the id here. i need to unify the identifiers of the extensions, but I have absolutely no time atm. In future python extension should have an identifier too. This means spell id would be org.albert.extension.python.spell and the datalocation therefore ~/.local/share/albert/org.albert.extension.python.spell/. On the other hand I am thinking about  removing this overprecise identifiers.

Reply:
fixed